RevenueCatUI/Templates/LinTemplate5View.swift

```diff
@@ -0,0 +1,316 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  LinTemplate5View.swift
+//
+//  Created by Nacho Soto.
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct LinTemplate5View: TemplateViewType {
+
+    let configuration: TemplateViewConfiguration
+
+    @State
+    private var selectedPackage: TemplateViewConfiguration.Package
+
+    @State
+    private var displayingAllPlans: Bool
+
+    @Environment(\.userInterfaceIdiom)
+    var userInterfaceIdiom
+    
+    @Environment(\.horizontalSizeClass)
+    var horizontalSizeClass
+    
+    @Environment(\.verticalSizeClass)
+    var verticalSizeClass
+
+    @Environment(\.locale)
+    var locale
+
+    @EnvironmentObject
+    private var introEligibilityViewModel: IntroEligibilityViewModel
+    @EnvironmentObject
+    private var purchaseHandler: PurchaseHandler
+
+    init(_ configuration: TemplateViewConfiguration) {
+        self._selectedPackage = .init(initialValue: configuration.packages.default)
+        self.configuration = configuration
+        self._displayingAllPlans = .init(initialValue: configuration.mode.displayAllPlansByDefault)
+    }
+
+    var body: some View {
+        self.content
+            .scrollableIfNecessary(enabled: self.configuration.mode.isFullScreen)
```

Comment:
if I have more then 2 offerings or a multi-line title, will 'purchase' button be pushed out of visible area?

Reply:
Currently, the primary content (buttons, title, and other texts) is attempted to always be visible by shrinking the top image. So while these elements fit into the window vertically, there should be no scrolling.

Reply:
@myeyesareblind I updated the UI layout to always keep the Subscribe button out of scroll